### DISCUSSION TITLE
3D MCNP Plotter

### DISCUSSION THREAD

**Discussion by SterlingButters** [idaholab/MontePy#366] [Ideas]
I recently developed a 3D plotter for MCNP. Before you get too excited though, it's more hacky than it sounds. Basically I use the PTRAC functionality to generate point clouds for each cell and perform surface reconstruction on the clouds. For fairly simple/small geometries this works appreciably well in my opinion. 

Just wondering if it might have a home here?

**Comment by MicahGale** [comment]
While it sounds like a great tool to have, it sounds more like it would be better off as a separate repository. 

I have been trying to curate the github tag: [mcnp](https://github.com/topics/mcnp), and I think making sure to tag good projects with that will be important going forward.

**Reply by SterlingButters** [reply]
Oh yeah that'd be pretty sweet
